src/app/frontend/deploy/createnamespace.html

```diff
@@ -19,7 +19,7 @@
     <h4 class="md-title" >Create a new namespace</h4>
     <p>The new namespace will be added to the cluster.</p>
     <!-- TODO(maciaszczykm): Missing detailed cluster info. -->
-    <form name="namespaceForm" ng-submit="ctrl.createNamespace()"
+    <form name="ctrl.namespaceForm" ng-submit="ctrl.createNamespace()"
```

Comment:
The validation did not work any longer. Not sure why. I fixed it by adding ctrl prefix....